makefile

```diff
@@ -30,19 +30,24 @@ VALUES :=  \
 	-DUSE_FULL_LL_DRIVER \
 	-DPREFETCH_ENABLE=1
 CFLAGS = $(MCU) $(VALUES) $(INCLUDES) -O2 -Wall -fdata-sections -ffunction-sections
-CFLAGS += -D$(TARGET)
+CFLAGS += -DUSE_$(TARGET)
 CFLAGS += -MMD -MP -MF $(@:%.bin=%.d)
-TARGETS := BOOTLOADER_PA2 BOOTLOADER_PB4
+
+# Targets by signal input pin:
+# PB4: iFlight
+# PA2: all others
```

Comment:
I think not only iflight use pb4, only few targets use pa2 most of the rest use pb4

Reply:
Got this from here: https://github.com/AlkaMotors/AM32_Bootloader_F051/blob/7f7e10a9381806019f77dd92aedc51eed0ae3c1c/Core/Src/main.c#L6

@AlkaMotors: Is this still correct?

Reply:
I think no, most of targets I submitted were using pb4 as input pin. Anyway, just a comment, rest part of code changes are nice!